opencensus/trace/ext/flask/flask_middleware.py

```diff
@@ -40,6 +40,9 @@
 SAMPLING_RATE = 'SAMPLING_RATE'
 TRANSPORT = 'TRANSPORT'
 SERVICE_NAME = 'SERVICE_NAME'
+JAEGER_EXPORTER_SERVICE_NAME = 'JAEGER_EXPORTER_SERVICE_NAME'
```

Comment:
I have a general feedback here: probably we want to refactor this code by decoupling the exporter logic and framework integration. How about extracting Jaeger/Zipkin/Ocagent specific thing from Django/Flask ext?

Reply:
@reyang seems like a great suggestion -- I was just leveraging what already existed. Can we capture this in a separate github issue?

Reply:
@flands #394 has been created to track this separately.

Reply:
@flands the branch is outdated, would you update it? I can help to merge.

Reply:
@reyang done!